massa-node/base_config/gas_costs/wasm_gas_costs.json

```diff
@@ -1,13 +1,13 @@
 {
-  "Wasm:Drop": 0,
-  "Wasm:GlobalGet": 4,
-  "Wasm:GlobalSet": 0,
-  "Wasm:I32Add": 126,
+  "Wasm:Drop": 8,
+  "Wasm:GlobalGet": 28,
+  "Wasm:GlobalSet": 26,
+  "Wasm:I32Add": 0,
   "Wasm:I32Const": 0,
```

Comment:
I think it should never be zero because someone could just make an infinite loop of just those instructions

Reply:
this needs a followup issue